### PR TITLE
Author a simple IDE perf test

### PR DESCRIPTION
**Pull request by rchande**
This test opens a C# solution, opens a file in the solution, and closes VS
when the CPU is idle. This test is based on the existing OpenSolution test.

cc @KevinH-MS @TyOverby @basoundr

### CODE REVIEW

**Comment by rchande** [comment]
Currently, running this requires you to do the following. I will work on automating these steps as well.

- [ ] Run parts of Perf-InstallRoslyn (closed) to install CodeMarkers in the RoslynPerf hive

- [ ] Build Roslyn and deploy it to the RoslynPerf hive

Furthermore:

- [ ] Tao needs to be able to customize the output directory of perf results

File: src/Test/Perf/projects/vsopencsharp/CSharpPerfSolutionLoad.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+
+<TaoTest xmlns="http://microsoft.com/schemas/VSLanguages/TAO">
```
Comment (jaredpar) [inline src/Test/Perf/projects/vsopencsharp/CSharpPerfSolutionLoad.xml]: I didn't think we had this exposed publicly.

Comment (rchande) [inline src/Test/Perf/projects/vsopencsharp/CSharpPerfSolutionLoad.xml]: Hm, good point. I'll find a way to target internal.

**Comment by rchande** [comment]
Closing in favor or https://github.com/dotnet/roslyn-internal/pull/794

**Comment by chrisaut** [comment]
What is roslyn-internal....and why does it exist? I thought everything was done out in the open?

**Comment by jaredpar** [comment]
@chrisaut a few pieces of our infrastructure are still closed and live in roslyn-internal.  This repo is closed because it refers to services that are only accessible internally and hence really have no value to the community.  There is no product code in roslyn-internal, just legacy infrastructure that we're slowly migrating into the open.

**Comment by chrisaut** [comment]
Fair enough I guess. Jusat surprised to see it that's all.

**Pull request closed without merging**